scout/server/blueprints/panels/templates/panels/panel_pdf_case_hits.html

```diff
@@ -0,0 +1,76 @@
+{% extends "report_base.html" %}
+
+{% block body %}
+<br>
+<div class="container">
+  <h4>Scout - Institute {{institute.display_name}} - case {{case.display_name}} - Gene panel variant hits</h4> - created on:&nbsp;<strong>{{report_created_at}}</strong><br><br>
+  {{ hits_panel() }}
+  <br>[END OF REPORT]<br><br>
+  <a href="https://clinical-genomics.github.io/scout" target="_blank" rel="noopener">clinical-genomics.github.io/scout</a>
+</div>
+{% endblock %}
+
+{% macro hits_panel() %}
+<table class="table table-sm">
+  <tr>
+    <th>Panel details</th>
+    <th>Case details</th>
+  </tr>
+  <tr>
+    <td>
+      Gene panel: <strong>{{panel.name_and_version}}</strong><br>
+      Last updated: <strong>{{ panel.date.strftime('%Y-%m-%d') }}</strong><br>
+      Database ID: {{ panel._id }}<br>
+      Description: {{ panel.description }}<br>
+      Number of genes: <strong>{{ panel.genes|length }}</strong><br>
+      Institute: {{ panel.institute }} <br>
+      Archived: {% if panel.is_archived %}<span class="badge bg-danger rounded-pill">True</span>{% else %}<span class="badge bg-secondary rounded-pill text-white">False</span>{% endif %}<br>
+    </td>
+    <td>
+      Case: <strong>{{case.display_name}}</strong><br>
+      Institute: <strong>{{institute.display_name}} - {{institute._id}}</strong><br>
+      Status: <strong>{{case.status}}</strong><br>
+      Analysis date: {{case.analysis_date}}<br>
+      Genome build: {{case.genome_build}}
+    </td>
+  </tr>
+</table>
+<br>
+<table>
+  <tr><th>SNV variants hits</th></tr>
```

Comment:
SNV and SVs would be the same for the standard panels.

Reply:
I'm just wondering, because this report can be also created from panels that were not used for the analysis of the case. I'm not saying that one should use it like that, but one could. In this situation perhaps it makes sense to have separate lists for SNVs and SVs?

Reply:
The question one wants to have the answer to is what genes are covered by this analysis.

This is all just a workaround for not having variant type in the panel definition, the way e.g. PanelApp do. Each gene should just have a tag with what variant types are considered clinical in that gene: SV, STR etc.

The ideal end result would be a join with a cohort level chanjo report, so that a gene that does not sequence well - gets low coverage on multiple analyses - is not on the list even if it is in the panel. But that is beyond us at the moment - chanjo2 perhaps? (Single case coverage is not quite enough either - homozygous deletion in part of a gene does not mean SNVs have not been considered). 

I can see very little difference between the SNV and SV lists otherwise, in default/master list panels or otherwise? 

The base use case would be for the panels that were used when running the case. I think it would make the most sense to attempt to give the list for the version of the panel that the user is at (typically latest; always versioned), but subtracted for genes that are not in the clinical list, much as we do when warning for searches on the variants page.